tools/test/devicesim/gnmi_user_manual.md

```diff
@@ -103,12 +103,26 @@ response: <
 ```
 
 ## Run the Subscribe command
+### Subscribe ONCE
 ```bash
 gnmi_cli -address localhost:10161 \
-       -proto "subscribe:<mode: 0, prefix:<>, subscription:<path: <elem: <name: 'openconfig-system:system'> elem: <name: 'clock' > elem: <name: 'config'> elem: <name: 'timezone-name'>>>>" \
```

Comment:
With or without the namespace should work

Reply:
Yes I know but it looks like when I remove it gnmi_cli gives me a path error (I don't think it is related to the code that I added). Let me take a closer look and then update you.

Reply:
I tested again with namespace and it works fine. Perhaps I had a typo when I was testing before. I will return the command to the original one.